tests/std/tests/P2465R3_standard_library_modules/test2.cpp

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import std.compat;
+
+#include <assert.h> // intentionally not <cassert>
+
+// INTENTIONALLY AVOIDED: using namespace std;
+
+void test_export_import_std() {
+    ::puts("std.compat: Testing `export import std;`.");
+
+    // Verify that std::vector is available:
+    const std::vector<int> v{10, 20, 30, 40, 50};
+    assert(v[2] == 30);
+
+    // Verify that `::operator new` is available:
+    const int* const ptr = new (std::nothrow) int{1729};
+    assert(ptr);
+    assert(*ptr == 1729);
+    delete ptr;
+
+    // Verify that the Sufficient Additional Overloads are available for std::sqrt():
+    assert(std::sqrt(25.0) == 5.0);
+    assert(std::sqrt(25.0f) == 5.0f);
+    assert(std::sqrt(25) == 5.0);
+    static_assert(std::is_same_v<decltype(std::sqrt(25.0)), double>);
+    static_assert(std::is_same_v<decltype(std::sqrt(25.0f)), float>);
+    static_assert(std::is_same_v<decltype(std::sqrt(25)), double>);
+}
+
+template <typename Void, typename... Args>
+struct IsGlobalHypotCallableImpl : std::false_type {};
+
+template <typename... Args>
+struct IsGlobalHypotCallableImpl<std::void_t<decltype(::hypot(std::declval<Args>()...))>, Args...> : std::true_type {};
+
+template <typename... Args>
+inline constexpr bool IsGlobalHypotCallable = IsGlobalHypotCallableImpl<void, Args...>::value;

```

Comment:
Style meganitpick: `inline` is odd in a non-header.

Reply:
Good catch, dropped. I think I was worrying about the new "warning C5260: the constant variable 'MEOW' has internal linkage in an included header file context, but external linkage in imported header unit context; consider declaring it 'inline' as well if it will be shared across translation units, or 'static' to express intent to use it local to this translation unit", but that occurs only when building a header unit; it isn't emitted for an ordinary source file.